src/vivarium_public_health/mslt/disease.py

```diff
@@ -96,33 +108,59 @@ class Disease:
 
     Parameters
     ----------
-    name
+    disease
         The disease name (referred to as `<disease>` here).
 
     """
 
-    def __init__(self, name):
-        self._name = name
-        self.configuration_defaults = {
-            self.name: {
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def configuration_defaults(self) -> Dict[str, Any]:
+        return {
+            self.disease: {
                 "simplified_no_remission_equations": False,
             },
         }
 
     @property
-    def name(self):
-        return self._name
+    def columns_created(self) -> List[str]:
+        columns = []
```

Comment:
Nit: maybe something like
`## disease, rate, scenario, when`
`column_params = [[self.disease], ["_S", "_C"], ["", "_intervention"] , ["", "_previous"]]`
`return [''.join(x) for x in itertools.product(*column_params)]`
would be nice here?

Reply:
I guess this is just a rearrangement of something that existed already, so maybe that's not on you to change here

Reply:
That's probably better, but I was very intentionally trying to change as little as possible.